utils/http_utils.go

```diff
@@ -36,7 +36,7 @@ func FormatURL(url string, entries ...string) string {
 	return fmt.Sprintf("%s%s", base, suffix)
 }
 
-func HttpExecute(httpClient *http.Client, logger *log.Logger, requestType string, requestURL string, rawPayload interface{}) (*http.Response, error) {
+func HttpExecute(httpClient *http.Client, logger *log.Logger, requestType string, requestURL string, user string, password string, rawPayload interface{}) (*http.Response, error) {
```

Comment:
this will break the calls in remote/client package, please do the changes to avoid that, also check if this code is called within docker plugin and k8s codes